backend/cmd/headlamp.go

```diff
@@ -815,12 +815,9 @@ func (c *HeadlampConfig) OIDCTokenRefreshMiddleware(next http.Handler) http.Hand
 		}
 
 		// skip if cluster is not using OIDC auth
-		oidcAuthConfig, err := kContext.OidcConfig()
-		if err != nil {
```

Comment:
I understand the need for removing the log but any specific reason to change the condition here?

Reply:
@yolossn Theoretically no but I thought it would be safer to confirm directly that the config was null, WDYT?

Reply:
If we are going to check for an error case we should ideally use the err check as it it provided by the function, In cases where the function doesn't return an error the null check makes sense.

Reply:
got it, will revert the condition back. tested once again and works as expected